### PR TITLE
Blackshield Equipment Change Ft. Trilly

### PR DESCRIPTION
**Pull request by EliteRiceFarmer**
Blackshield Equipment Change (With assistance of Trilly)

Changes:
!1 Replaced: Tactical knife were replaced with Blackshield version.

!2 Added: New Saber and Scabbard for the Commander and the Sergeant. 10% Spawn chance for Silencer (Trilly's Request.)

!3 Removed: Excessive, unneeded codes in Blackshield's locker. (Items that never really spawn.)

!4 Repaired: Typo in Blackshield accessories code as well repaired the missing new and src for a few items in the locker. 
(Without those they won't spawn.)

### CODE REVIEW

File: code/game/objects/structures/crates_lockers/closets/secure/militia.dm [hunk 1]
```diff
@@ -9,36 +9,38 @@
...
-	if(prob(50))
-		new /obj/item/storage/backpack/satchel/ironhammer(src)
-	else
-		new /obj/item/storage/backpack/militia(src)
+	new /obj/item/melee/telebaton(src)
+	new /obj/item/storage/pouch/baton_holster(src)
 	new /obj/item/storage/box/commanderuniform(src)
-	if(prob(5))
-		new /obj/random/rations/crayon(src)
+if(prob(10))
```
Comment (Trilbyspaceclone) [inline code/game/objects/structures/crates_lockers/closets/secure/militia.dm:42]:
```suggestion
	if(prob(10))
```

File: code/game/objects/structures/crates_lockers/closets/secure/militia.dm [hunk 2]
```diff
@@ -9,36 +9,38 @@
...
 	new /obj/item/storage/box/trackimp(src)
-	if(prob(50))
-		new /obj/item/storage/backpack/satchel/ironhammer(src)
-	else
-		new /obj/item/storage/backpack/militia(src)
+	new /obj/item/melee/telebaton(src)
+	new /obj/item/storage/pouch/baton_holster(src)
 	new /obj/item/storage/box/commanderuniform(src)
-	if(prob(5))
-		new /obj/random/rations/crayon(src)
```
Comment (Trilbyspaceclone) [inline code/game/objects/structures/crates_lockers/closets/secure/militia.dm:41]: >:T why are you removing the crayons

File: code/game/objects/structures/crates_lockers/closets/secure/militia.dm [hunk 3]
```diff
@@ -70,16 +75,16 @@
...
+	new /obj/item/storage/backpack/militia(src)
+	new /obj/item/storage/pouch/large_generic(src)
 	new /obj/item/storage/pouch/ammo(src)
 	new /obj/item/storage/firstaid/ifak(src)
+	new /obj/item/melee/telebaton(src)
+	new /obj/item/storage/pouch/baton_holster(src)
 	new /obj/item/storage/box/sergeantuniform(src)
-	if(prob(5))
-		new /obj/random/rations/crayon(src)
+if(prob(10))
```
Comment (Trilbyspaceclone) [inline code/game/objects/structures/crates_lockers/closets/secure/militia.dm:86]:
```suggestion
	if(prob(10))
```

File: code/game/objects/structures/crates_lockers/closets/secure/militia.dm [hunk 4]
```diff
@@ -93,38 +98,37 @@
...
+	new /obj/item/storage/pouch/large_generic(src)
 	new /obj/item/storage/pouch/ammo(src)
-	new /obj/item/storage/pouch/baton_holster(src)
 	new /obj/item/storage/firstaid/ifak(src)
+	new /obj/item/melee/telebaton(src)
+	new /obj/item/storage/pouch/baton_holster(src)
 	new /obj/item/storage/box/trooperuniform(src)
-	if(prob(5))
-		new /obj/random/rations/crayon(src)
+if(prob(10))
```
Comment (Trilbyspaceclone) [inline code/game/objects/structures/crates_lockers/closets/secure/militia.dm:130]:
```suggestion
	if(prob(10))
```

File: code/game/objects/structures/crates_lockers/closets/secure/militia.dm [hunk 5]
```diff
@@ -137,14 +141,16 @@
 	new /obj/item/ammo_magazine/speed_loader_rifle_75(src)
 	new /obj/item/ammo_magazine/speed_loader_rifle_75(src)
 	new /obj/item/ammo_magazine/speed_loader_rifle_75(src)
-	new /obj/item/ammo_magazine/speed_loader_rifle_75(src)
```
Comment (Trilbyspaceclone) [inline code/game/objects/structures/crates_lockers/closets/secure/militia.dm:140]: why are you removing this one?

File: code/game/objects/structures/crates_lockers/closets/secure/militia.dm [hunk 6]
```diff
@@ -155,41 +161,39 @@
...
 	new /obj/item/storage/firstaid/combat(src)
-	if(prob(25))
-		new /obj/item/storage/firstaid/blackshield/large(src)
+	new /obj/item/storage/firstaid/blackshield/large(src)
+	new /obj/item/melee/telebaton(src)
+	new /obj/item/storage/pouch/baton_holster(src)
 	new /obj/item/storage/box/trooperuniform(src)
-	if(prob(5))
-		new /obj/random/rations/crayon(src)
+if(prob(10))
```
Comment (r4d6) [inline code/game/objects/structures/crates_lockers/closets/secure/militia.dm:175]:
```suggestion
	if(prob(10))
```

Comment (Trilbyspaceclone) [reply code/game/objects/structures/crates_lockers/closets/secure/militia.dm:175]: did i miss this one?

Comment (r4d6) [reply code/game/objects/structures/crates_lockers/closets/secure/militia.dm:175]: You did.

File: code/game/objects/structures/crates_lockers/closets/secure/militia.dm (hunk 5 above)
Comment (EliteRiceFarmer) [reply code/game/objects/structures/crates_lockers/closets/secure/militia.dm:140]: The pouch can only hold 3.

File: code/game/objects/structures/crates_lockers/closets/secure/militia.dm (hunk 2 above)
Comment (EliteRiceFarmer) [reply code/game/objects/structures/crates_lockers/closets/secure/militia.dm:41]: Unga bad

File: code/game/objects/structures/crates_lockers/closets/secure/militia.dm (hunk 5 above)
Comment (Trilbyspaceclone) [reply code/game/objects/structures/crates_lockers/closets/secure/militia.dm:140]: you load 1 clip into gun and have 3 in pounch

File: code/game/objects/structures/crates_lockers/closets/secure/militia.dm [hunk 7]
```diff
@@ -137,14 +141,16 @@
 	new /obj/item/ammo_magazine/speed_loader_rifle_75(src)
 	new /obj/item/ammo_magazine/speed_loader_rifle_75(src)
 	new /obj/item/ammo_magazine/speed_loader_rifle_75(src)
```
Comment (Trilbyspaceclone) [inline code/game/objects/structures/crates_lockers/closets/secure/militia.dm:143]:
```suggestion
	new /obj/item/ammo_magazine/speed_loader_rifle_75(src)
	new /obj/item/ammo_magazine/speed_loader_rifle_75(src)
```

**Comment by Kazkin** [comment]
Randomized side arms are to remain, PR will be paused until the chance of getting a lamia instead of a colt is re-added.

**Comment by EliteRiceFarmer** [comment]
> Randomized side arms are to remain, PR will be paused until the chance of getting a lamia instead of a colt is re-added.

Trilby, me and r4d6 already discussed this in the temporary Discord along with other Blackshield players.
Would you like to tell me why you want the 50. Option?

I myself is a Blackshield main and i don't find the reason of logistical nightmare... A good idea.

**Comment by r4d6** [comment]
> > Randomized side arms are to remain, PR will be paused until the chance of getting a lamia instead of a colt is re-added.
> 
> Trilby, me and r4d6 already discussed this in the temporary Discord along with other Blackshield players.
> Would you like to tell me why you want the 50. Option?
> 
> I myself is a Blackshield main and i don't find the reason of logistical nightmare... A good idea.

Honestly I don't mind it going either way.
Beside, troopers getting the Lamia instead of the colt give them a reason to ask for different ammo.

**Comment by Kazkin** [comment]
Randomized equipment is coooooooool so having a sometimes varied loadout gives some interesting round starts. That is why originally we had a much more varied system, simply having the potential to get a more powerful side arm isn't a downgrade, removing the chance at all is.

**Comment by Kazkin** [comment]
Trilby will re0add the missing RNG spawners, this will be merged and updated before going live.

**Pull request merged**